extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -210,11 +346,31 @@ export class AmpConsent extends AMP.BaseElement {
     if (!this.consentUIRequired_[instanceId]) {
       return;
     }
+
+    const promptUI = this.consentConfig_[instanceId]['promptUI'];
+    const element = this.getAmpDoc().getElementById(promptUI);
+    this.consentUI_[instanceId] = element;
+
+    if (!this.revokeUI_ && this.consentConfig_[instanceId]['revokeUI']) {
+      this.revokeUI_ = this.getAmpDoc().getElementById(
+          this.consentConfig_[instanceId]['revokeUI']);
+    }
+
+    // Get current consent state
+    this.consentStateManager_.getConsentInstanceState(instanceId)
+        .then(state => {
+          if (state == CONSENT_ITEM_STATE.UNKNOWN) {
+
+            this.notificationUiManager_.registerUI(
```

Comment:
Do we not care about order of consents shown? Since this is driven by `Object.keys` it is not deterministic. Not sure it matters or not

Reply:
At least for now, we don't plan on ordering consents.